Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM mcr.microsoft.com/vscode/devcontainers/rust:0-1
 
 RUN rustup target add wasm32-unknown-unknown
 
-ENV DENO_INSTALL=/usr/local
 # crates.io index update fails without this (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=918854)
 ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
 
-RUN curl -fsSL https://deno.land/x/install/install.sh | sh
-RUN curl -fsSL https://dprint.dev/install.sh | sh
+USER vscode
 
-RUN export PATH=/usr/local/cargo/bin:$PATH
-RUN export DPRINT_INSTALL="/home/vscode/.dprint"
-RUN export PATH="$DPRINT_INSTALL/bin:$PATH"
+RUN /bin/bash -c 'curl -fsSL https://deno.land/x/install/install.sh | sh \
+  && curl -fsSL https://dprint.dev/install.sh | sh \
+  && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash \
```

Comment:
this is only used for `cspell`.

An alternative installation method, could be to use node from NodeSource

```
curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
sudo apt-get install -y nodejs
```

Thoughts?

Reply:
I suppose we still need to set up Node in order for Prettier and other devcontained extensions to work properly. [I stackoverflowed this](https://stackoverflow.com/questions/67959838/most-efficient-docker-image-with-both-node-deno-installed) over a year ago. The answer is subpar. @tjjfvi any recommendations?

Reply:
Not familiar with docker, but the setup seems fine to me 🤷

Reply:
I decided to use `https://deb.nodesource.com/setup_18.x` so we don't need to create a non-root user to run `nvm`.

Note: this would be an alternative to setup nvm as a root or non-root user https://github.com/microsoft/vscode-dev-containers/blob/v0.195.0/containers/javascript-node/.devcontainer/base.Dockerfile#L10